.env.example

```diff
@@ -36,8 +36,6 @@ BRP_API_CLIENT_MP_REST_URL=http://localhost:5010/haalcentraal/api/brp
 # Het RSIN van de Niet-natuurlijk persoon zijnde de organisatie die de zaak heeft gecreeerd.
 # Dit moet een geldig RSIN zijn van 9 nummers en voldoen aan https://nl.wikipedia.org/wiki/Burgerservicenummer#11-proef
 BRON_ORGANISATIE_RSIN=123443210
-# Not used when using the BRP mock
```

Comment:
I would it in here actually because this file is where we document all environment variables in ZAC, also the ones that are optional. This `.env.example` file is not used, it is just documentation. It overlaps partly with the Helm Chart but I think that's not too bad because they have different audiences you could say.

Reply:
All properties are now in .env.example